README.md

```diff
@@ -1,45 +1,40 @@
 # mathjax-node [![Build Status](https://travis-ci.org/mathjax/MathJax-node.svg?branch=develop)](https://travis-ci.org/mathjax/MathJax-node)
 
-This repository contains a library that provides an API to call [MathJax](https://github.com/mathjax/mathjax) from
-NodeJS programs.  The API converts individual math
-expressions (in any of MathJax's input formats) into HTML (with CSS), SVG or MathML code.
-
-See the comments in the individual files for more details.
-
-The `bin` directory contains a collection of command-line programs for
-converting among MathJax's various formats.  These can be used as examples
-of calling the MathJax API.
+This repository contains a library that provides an API to call [MathJax](https://github.com/mathjax/mathjax) from Node.js programs. The API converts individual math expressions (in any of MathJax's input formats) into HTML (with CSS), SVG or MathML code.
 
 Use
 
-    npm install mathjax-node
+```
+npm install mathjax-node
+```
 
 to install mathjax-node and its dependencies.
 
-
-**Note:** 
+**Note:**
 
 mathjax-node requires Node.js v4 or later.
 
-
 **Breaking Changes in v1.0:**
 
 mathjax-node v1.0 dropped the following features that were present in earlier pre-releases.
 
-* `lib/mj-page.js` (API for processing HTML-fragments) and related CLI tools
-* speech-rule-engine integration
-* PNG generation
+- `lib/mj-page.js` (API for processing HTML-fragments) and related CLI tools
+- speech-rule-engine integration
+- PNG generation
+- CLI tools in `bin/`
```

Comment:
I would add a comment here about the name change from `mj-single.js` to `main.js`, since people will have been loading that explicitly.